rosidl_parser/rosidl_parser/definition.py

```diff
@@ -448,7 +448,7 @@ def get_comment_lines(self):
         ]
         lines = []
         for comment in comments:
-            lines.extend(comment.splitlines())
+            lines.extend([x.strip() for x in comment.splitlines()])
```

Comment:
Stripping the whitespace might be ok for comments, but it looks like there's a larger problem with parsing string literals. I've opened an alternative to this PR in #688.